shared/constants/fs.js

```diff
@@ -182,6 +182,21 @@ const itemStylesTeamTlf = memoize((teamName: string) => ({
   textType: folderTextType,
 }))
 
+export const humanReadableFileSize = (meta: Types.PathItemMetadata) => {
+  const kib = 1024
+  const mib = kib * kib
+  const gib = mib * kib
+  const tib = gib * kib
+
+  if (!meta) return ''
+  const size = meta.size
+  if (size >= tib) return `${Math.round(size / tib)}TB`
+  if (size >= gib) return `${Math.round(size / gib)}GB`
+  if (size >= mib) return `${Math.round(size / mib)}MB`
+  if (size >= kib) return `${Math.round(size / kib)}KB`
+  return '' + size
```

Comment:
This should be `` `${size}B` ``

Reply:
I think there should also be a space before the unit? The popup design has space but not in the file info screen.

Reply:
Yup, sure.